spec/01-unit/01-db/01-schema/13-cluster_status_spec.lua

```diff
@@ -0,0 +1,42 @@
+require "spec.helpers" -- initializes 'kong' global for plugins
+local Entity = require "kong.db.schema.entity"
+local cluster_status_definition = require "kong.db.schema.entities.cluster_status"
```

Comment:
This variable could be renamed to `cluster_status_schema`. The term "definition" is usually reserved for the table we use to create a schema (`schema = Schema.new(definition)`).